app/Resources/assets/scss/admin.scss

```diff
@@ -0,0 +1,26 @@
+@import "~bootswatch/flatly/variables";
+@import "~eonasdan-bootstrap-datetimepicker/src/sass/bootstrap-datetimepicker-build.scss";
+@import "~bootstrap-tagsinput/src/bootstrap-tagsinput.css";
```

Comment:
When a CSS file is a dependency of some JS library, I usually like to require the CSS file right in our `.js` file, right next to where I require the related JS file. Later, if we stopped needing the bootstrap-datetimepicker JS file, we would also remove the CSS require right next to it.